### DISCUSSION TITLE
Interacive Map

### DISCUSSION THREAD

**Discussion by ARUIZI01** [opengeos/segment-geospatial#276] [Q&A] [answered]
Hello, I was making use of the code in Google Colab, but today the interactive map doesn't come out.

**Comment by giswqs** [comment, marked as answer]
Run `pip install ipyleaflet==0.18.2` to download ipyleaflet. 

https://github.com/googlecolab/colabtools/issues/4540

**Reply by ARUIZI01** [reply]
Thak you so much
